.travis.yml

```diff
@@ -5,6 +5,10 @@ language: go
 services:
   - docker
 
+script:
+  - go install github.com/onsi/ginkgo/ginkgo
```

Comment:
from their github page:

```
go get -u github.com/onsi/ginkgo/ginkgo  # installs the ginkgo CLI
go get -u github.com/onsi/gomega/...     # fetches the matcher library
```

Reply:
that's exactly what we want to avoid. we already have the code checked in.